api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -61,6 +61,13 @@
  *   <li>to cancel the LRA context when the method returns particular HTTP
  *       status codes</li>
  * </ul>
+ * 
+ * The LRA operations are performed before the execution proceeds to the LRA
+ * annotated business method. If any of the LRA operations cannot complete the
+ * implementation MUST return one of the defined HTTP status codes which means
+ * that the business method is not invoked. For the definition of the allowed
+ * status codes and condition in which they are returned please see the 
+ * specification document.
```

Comment:
I wonder if cross-referencing from the javadoc makes sense. The section numbers and titles are changed easily. For me the current text is better.

Reply:
Yes, I agree that keeping this information in Javadoc up to date may be problematic. The Javadoc itself cannot be enough to understand the spec. Otherwise, we don't need the specification document at all.